pyproject.toml

```diff
@@ -59,14 +59,69 @@ script-files = ["scripts/download_gleam.py", "scripts/make_flat_spectrum_eor.py"
 
 [tool.setuptools_scm]
 
-[tool.isort]
-profile = "black"
-
 [darglint]
 docstring_style = "numpy"
 
-[tool.black]
-skip_magic_trailing_comma = true
-
 [tool.pytest.ini_options]
 addopts = "--ignore=scripts"
+
+[tool.ruff.lint]
+select = [
+    # pycodestyle
+    "E",
+    # pycodestyle warnings
+    "W",
+    # Pyflakes
+    "F",
+    # pydocstyle
+    "D",
+    # pyupgrade
+    "UP",
+    # flake8-bugbear
+    "B",
+    # flake8-builtins
+    "A",
+    # flake8-comprehensions
+    "C4",
+    # pep8-naming
+    "N",
+    # flake8-simplify
+    "SIM",
+    # isort
+    "I",
+    # McCabe complexity. Consider for the future
+    # "C90"
```

Comment:
nice thing is the toml format allows the comments to be inline, so these can all be compressed